### DISCUSSION TITLE
FluidProperties doesn't work

### DISCUSSION THREAD

**Discussion by SomnusYu** [idaholab/moose#22705] [Q&A General] [answered]
Hi all,

After I updated my moose version, the mistake occurred in my previous case of my application:
![image](https://user-images.githubusercontent.com/118275988/201924375-7be448db-6985-4125-8ba3-b147e875a6bd.png)

I noticed that MOOSE has changed /Modules/FluidProperties/* to /FluidProperties/*, but why does the app in the Modules using FluidProperties not report an error, but my own app does?

Could anyone kindly help me to figure it out?

Best regards!
Somnus

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did you recompile the app as well after updating or is it the old executable?
The new syntax is indeed [FluidProperties] instead of [Modules/FluidProperties]

Guillaume

**Reply by SomnusYu** [reply]
Yes, I have recompiled the app. I know that the new syntax is [FluidProperties], which is also used in my input file. Should I uninstall the MOOSE and then reinstall it?

**Reply by GiudGiud** [reply]
You could git clean -xfd in the moose folder then re-make

Be careful that will delete ALL files that are not saved!! So please make sure to commit them first if you have modified MOOSE

**Reply by SomnusYu** [reply]
Ok, I will try it. Thank you very much!
